app/src/main/res/layout/fragment_second.xml

```diff
@@ -9,27 +9,94 @@
 
     <!-- TODO: Update blank fragment layout -->
 
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="413dp"
+        android:layout_height="339dp"
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginTop="4dp"
+        android:contentDescription="@string/imageavatar"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:src="@drawable/vinagre" />
+
     <TextView
-        android:id="@+id/text3"
+        android:id="@+id/fullname"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
-        android:layout_marginLeft="32dp"
-        android:layout_marginTop="32dp"
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginTop="352dp"
         android:background="#F5F5F8"
-        android:text="@string/text3"
+        android:text="@string/fullname"
         app:layout_constraintStart_toStartOf="parent"
```

Comment:
Full name should be constraint to the photo.
Try this
`<TextView
        android:id="@+id/fullname"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_margin="24dp"
        android:background="#F5F5F8"
        android:text="@string/fullname"
        app:layout_constraintStart_toStartOf="@+id/imageView"
        app:layout_constraintBottom_toBottomOf="@+id/imageView" />`